services/minespace-web/package.json

```diff
@@ -123,7 +132,6 @@
   },
   "optionalDependencies": {
     "enzyme-adapter-react-16": "1.14.0",
-    "jest": "24.8.0",
     "jest-canvas-mock": "2.1.0",
     "jest-environment-jsdom": "24.8.0",
     "jest-environment-jsdom-global": "1.2.0",
```

Comment:
I think the format scripts just below here (ln 150-152) and the equivalent bits in the core-web file are all that remains.